models/reviews-models.js

```diff
@@ -55,3 +55,28 @@ exports.fetchReviews = () => {
       return rows;
     });
 };
+
+exports.addReviewComment = (review_id, username, body) => {
+  if (!username || !body) {
+    console.log("inside first models");
+    return Promise.reject({ status: 400, msg: "Route not found" });
```

Comment:
as mentioned in my comment in `app.test.js` this error `msg` doesn't make sense for this error